.devcontainer/typescript/devcontainer.json

```
@@ -0,0 +1,24 @@
+{
+	"name": "TypeScript",
+	"dockerComposeFile": [
+	  "./docker-compose.yml"
+	],
```

Merge Docker Compose to override project mount and service entry points as in the Java version

```suggestion
	"dockerComposeFile": [
	  "./docker-compose.yml",
	  "../docker-compose-common.yml"
	],
```